crates/tabby-common/src/index/mod.rs

```diff
@@ -104,7 +104,7 @@ impl IndexSchema {
         let field_updated_at = builder.add_date_field(FIELD_UPDATED_AT, INDEXED | STORED);
         let field_failed_chunks_count =
             builder.add_u64_field(FIELD_FAILED_CHUNKS_COUNT, INDEXED | FAST | STORED);
-        let field_attributes = builder.add_text_field(FIELD_ATTRIBUTES, STORED);
+        let field_attributes = builder.add_json_field(FIELD_ATTRIBUTES, FAST | STORED);
```

Comment:
refer line 110 to set index options using `JsonObjectOptions` (the config can be same)